.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
+config/**/*.config.php
```

Comment:
??

Reply:
Look at the files :)
There is no yml file anymore, there are `config/.../jira.config.php.dist` files which will be replaced by jira.config.php` which i dont want to be checked into version control

Reply:
Ah, jetzt dist files. Verstehe.